modules/dicehub/dbclient/image.go

```diff
@@ -54,15 +54,6 @@ func (client *DBClient) GetImagesByRelease(releaseID string) ([]Image, error) {
 	return images, nil
 }
 
-// GetImagesByID 根据 imageID 获取镜像
-func (client *DBClient) GetImageByID(imageID int64) (*Image, error) {
-	var image Image
-	if err := client.Where("id = ?", imageID).Find(&image).Error; err != nil {
-		return nil, err
-	}
-	return &image, nil
-}
-
 // GetImageByImage 根据 image 获取镜像
 func (client *DBClient) GetImageByImage(image string) (*Image, error) {
```

Comment:
Migrate all image related functions together.

Reply:
it is used by release provider. not image provider